src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1131,14 +1131,12 @@ def matrix_stratification_values(self, file: api.File) -> JSON:
                     contents['donors'].append(donors)
                 organ = stratum.get('organ')
                 if organ is not None:
-                    organ = sorted(organ)
-                    contents['samples'].append(
+                    organ = ', '.join(sorted(organ))
```

Comment:
I assume the space after `,` is intentional?

Reply:
Thank you for pointing this out. Although the `specimens` inner entity field `organ` is string, joining the (possibly multiple) organ values provided by the CGM's metadata into one string is incorrect. Instead separate `specimens` inner entity should be added for each organ value.